generic/ltp.py

```diff
@@ -79,14 +82,19 @@ def setUp(self):
         dist = distro.detect()
         self.args = self.params.get('args', default='')
         self.mem_leak = self.params.get('mem_leak', default=0)
+        self.peer_public_ip = self.params.get("peer_public_ip", default="")
+        self.peer_user = self.params.get("peer_user", default="root")
+        self.peer_password = self.params.get("peer_password", default=None)
+        self.two_host_configuration = self.params.get("two_host_configuration",
+                                                      default=False)
 
-        deps = ['gcc', 'make', 'automake', 'autoconf', 'psmisc']
+        deps = ['gcc', 'make', 'automake', 'autoconf', 'psmisc', 'vsftpd']
```

Comment:
having network related package should only be installed when a network test need to be requested  to run 

something like 

```
if "-f net" in self.args:
  deps.extend("vsftpd")
  
```